src/components/include/DefaultImportModalDialog.jsx

```diff
@@ -166,12 +200,53 @@ class DefaultImportModalDialog extends React.Component {
         }));
     };
 
+    handleDefaultMaterials = async () => {
+        const { selectedMaterial, files } = this.state;
+
+        if (!selectedMaterial) {
+            console.error("No material selected");
+            return;
+        }
+        const config = selectedMaterial.value;
+
+        const newFile = {
+            id: config.id || files.length,
+            fileName: config.name || "Not available",
+            format: "json",
+            text: JSON.stringify(config) || "Not available",
+            lastModified: this.formatDate(new Date()),
+        };
+
+        this.setState({ files: [...files, newFile] });
+        this.setState({ selectedMaterial: null });
```

Comment:
Let's move to open

Reply:
It's needed here, because it cleans the selected material when we added it to files list, then we might want to pick another.